src/vector/geometry.rs

```diff
@@ -480,9 +514,45 @@ impl Debug for GeometryRef<'_> {
     }
 }
 
+/// Invocation options for [`Geometry::make_valid_ex`].
+#[derive(Debug, Clone, Default)]
+pub enum MakeValidOpts {
```

Comment:
I feel (not that strongly, though) we should allow the caller to pass in their own options in case a future version of GDAL adds an option we don't know about yet,

Reply:
Thoughts on how that might be reflected in the implementation while not falling back to everything accepting `CslStringList` (which I think misses an opportunity for safer usage).